core/v0/api/core.yaml

```diff
@@ -1488,7 +1488,7 @@ components:
         additional_description:
           $ref: '#/components/schemas/Descriptor'
     Catalog:
-      description: Describes a BPP catalog
+      description: Describes the products or services offered by a BPP. This is typically sent as the response to a search intent from a BAP. The payment terms, offers and terms of fulfillment supported by the BPP can also be included here.<br>This has properties like bbp/descriptor,bbp/categories,bbp/fulfillments,bbp/payments,bbp/offers,bbp/providers and exp<br>This is used in the following situations.<br><ul><li>This is typically used in the discovery stage when the BPP sends the details of the products and services it offers as response to a search intent from the BAP. </li></ul>
```

Comment:
Please expand the description to also include ability or BAP or BG to cache a BPP catalog based on TTL (time to live). Also expand the description to include the hierarchical nature of this, use of categories, etc.